mod_reforged/hooks/items/weapons/throwing_axe.nut

```diff
@@ -5,4 +5,13 @@
 		create();
 		this.m.Reach = 0;
 	}
+
+	o.onEquip = function()
+	{
+		this.weapon.onEquip();
+
+		this.addSkill(::MSU.new("scripts/skills/actives/throw_axe", function(o) {
+			o.m.FatigueCost -= 2;
```

Comment:
So, it seems we want -2 fat cost on throw_axe by default, and then +1 on the heavy version.